.pipelines/pipeline.user.windows.yml

```diff
@@ -75,6 +84,23 @@ build:
             - 'modules\ImageResizer\sk\ImageResizer.resources.dll'
             - 'modules\ImageResizer\tr\ImageResizer.resources.dll'
             - 'modules\ImageResizer\zh-Hans\ImageResizer.resources.dll'
+            - 'modules\launcher\cs\PowerLauncher.resources.dll'
```

Comment:
can we do something like: 'modules\launcher\*\PowerLauncher.resources.dll'

Reply:
I'll have to try but it might be possible. If possible then a lot of it can be simplified here (maybe even modules\*.resources.dll or something of that sort if it works.